include/hello_log_ids.hrl

```diff
@@ -0,0 +1,90 @@
+-define(LOGID99, {ffffffffffffffffffffffffffffffff, 999999}). %placeholder
+
+%% since erlang atoms are quoted if they begin with a number
+%% we just take UUIDs which begin with a letter. Hence we have
+%% no quoting in journald (like other message IDs).
```

Comment:
The lager_journald_backend wouldn't like that. Apart from that I would use binaries too ...